### PR TITLE
refactor about activity code and layout with camelcase convention

### LINKED ISSUE #2875: refactor About us Activity layout and activity code
**Describe the bug**

Currently the about us layout has unnecessary views and groups, simplify them to use a constraint layout and use camelCase for naming variables.

**Expected behavior**
Standard common casing and layout should be properly written with no extra views

**Logs**

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by codedsun**
Fixed #2875 : refactor about activity code and layout with camelcase convention

Changes: Followed camelCase and removed unnecessary views, simplified view hierarchy with constraint layout

Screenshots of the change:

### CODE REVIEW

**Comment by codedsun** [comment]
@yashk2000 please review

**Comment by codedsun** [comment]
@iamareebjamal Please check

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/AboutActivity.java [hunk 1]
```diff
@@ -28,8 +28,8 @@
 import org.fossasia.phimpme.gallery.util.CustomTabService;
 import org.fossasia.phimpme.utilities.ActivitySwitchHelper;
 
-/** Created by Jibo on 02/03/2016. */
-public class AboutActivity extends ThemedActivity {
+/** Created by codedsun on 26/9/2019. */
```
Comment (yashk2000) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/AboutActivity.java]: irrelavant change

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -269,17 +269,14 @@
                 <category android:name="android.intent.category.DEFAULT" />
             </intent-filter>
         </activity>
+
         <activity
             android:name=".gallery.activities.AboutActivity"
             android:configChanges="orientation|screenSize"
             android:label="@string/about"
             android:theme="@style/Theme.AppCompat.NoActionBar">
-            <intent-filter>
```
Comment (yashk2000) [inline app/src/main/AndroidManifest.xml]: Is removing this really needed?

**Comment by yashk2000** [comment]
Please provide screenshots of the change if any.

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (codedsun) [reply app/src/main/AndroidManifest.xml]: @yashk2000 Why is this added?

**Comment by codedsun** [comment]
image


@yashk2000 Review

**Comment by codedsun** [comment]
@iamareebjamal Okay, will update!

**Comment by codedsun** [comment]
@iamareebjamal @yashk2000 Please get this done!

**Comment by codedsun** [comment]
@iamareebjamal Please get this merged!

**Comment by codedsun** [comment]
@iamareebjamal @yashk2000 Please merge

**Pull request merged**